utils/src/ooinstall/oo_config.py

```diff
@@ -36,19 +36,24 @@ def __init__(self, **kwargs):
         self.public_ip = kwargs.get('public_ip', None)
         self.public_hostname = kwargs.get('public_hostname', None)
         self.connect_to = kwargs.get('connect_to', None)
+        self.run_on = kwargs.get('run_on', None)
 
         # Should this host run as an OpenShift master:
         self.master = kwargs.get('master', False)
 
         # Should this host run as an OpenShift node:
         self.node = kwargs.get('node', False)
+
+        # Should this host run as an HAProxy:
+        self.ha_proxy = kwargs.get('ha_proxy', False)
```

Comment:
I'm wondering if we should rename this new setting to just `master_loadbalancer` or just `master_lb`.  I'd hate to have a `ha_proxy` setting and then our playbooks swap it out for something else.  @dgoodwin, thoughts?